lib/config/rules/node.js

```diff
@@ -42,8 +52,12 @@ module.exports = {
   'node/no-unpublished-bin': 'error',
   // Disallow require()s for files that are not published.
   'node/no-unpublished-require': 'off',
-  // Disallow unsupported ECMAScript features on the specified version.
-  'node/no-unsupported-features': 'off',
+  // disallow unsupported ECMAScript built-ins on the specified version
+  'node/no-unsupported-features/es-builtins': 'off',
+  // disallow unsupported ECMAScript syntax on the specified version
+  'node/no-unsupported-features/es-syntax': 'off',
+  // disallow unsupported Node.js built-in APIs on the specified version
+  'node/no-unsupported-features/node-builtins': 'off',
```

Comment:
I feel like this one should be on

Reply:
We had all the other `node/no-unsupported-features/*` disabled, so i thought we'd want this disabled as well. enabling now.